### DISCUSSION TITLE
Maker Pi RP2040

### DISCUSSION THREAD

**Discussion by Tourniquet63** [BIPES/BIPES#198] [General]
Hi,

Thanks so much for making this site.  I am an educator and have had a rought couple years transitioning to chromebooks while teaching microprocessors, so I am particularly thankful for this site that has the promise of programming a Pi Pico right from the browser.

I have got some RP Pi Pico's to work but I have another classt that mainly run [MakerPi 2040's ](https://www.cytron.io/p-maker-pi-rp2040-simplifying-robotics-with-raspberry-pi-rp2040).  I don't know if this board will run under the Pi Pico selection or if it would need some more support before working.

**Comment by jetannenbaum** [comment]
Hi,

Use this site: https://bipes.net.br/pico/ui/

It has support for the Pico, Pico w, and the maker pi rp2040 boards

Also, I built this site for a talk I gave for Cody Savvy: https://jetannenbaum.github.io/Bipes4RP2040/.  It contains a set of lessons for teaching Picos with BIPES 

Jim

**Comment by Tourniquet63** [comment]
Wow - what a dream!  Thank you - I can't wait to get into this.

**Comment by mperino** [comment]
I'm working on Bipes with Makerpi 2040's for our FRC Robotics Team to teach younger kids about robots.  Seems to mostly work so far.

**Comment by mperino** [comment]
@Tourniquet63 You might find what I'm doing interesting. 

Our Highschool Robotics kids (Team 4160), are going to host several workshops for middle school kids and girlscouts to teach some robotics basics.  We wanted to have a cheap and capable robot to teach programming to kids of that age group.

Meet Scout RoBuc:
https://www.printables.com/model/959049-lego-compatible-robuc-robot-bucaneer-body

Depending on what sensors you pair with it.  Price is $36-$76 before shipping worth of parts to make 1 robot.  Since some parts come in 2 packs, unit cost is $29.25-$56.75

Plus some of the parts are ones you might have lying around from other projects.

Demo:
https://www.youtube.com/watch?v=4y50CAeZjC8&sttick=0

**Comment by jetannenbaum** [comment]
Interesting.  I use off the shelf components for my "MakerPi Bot."  Take a look at https://jetannenbaum.github.io/STEMatUDL/robotCar/makerPiRP2040Bot/

I've bulk ordered the parts from Cytron.io and gotten pretty good pricing.  In the past the kit ran about $20 - $25 dollars.
